src/components/common/button/Button.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+interface ButtonProps {
+  onClick: () => void;
+  variant?: 'solid' | 'light' | 'ghost' | 'outline' | 'gray' | 'red';
+  size?: 'sm' | 'md' | 'lg';
+  children: React.ReactNode;
+}
+
+// 변형 스타일
+const variantStyles = {
+  solid: 'bg-primary-01 text-white-01',
+  light: 'bg-primary-03 text-primary-01',
+  ghost: 'border border-primary-01 bg-white-01 text-primary-01',
+  outline: 'border border-gray-02 text-black-02',
+  gray: 'bg-gray-02 text-white-01 cursor-not-allowed',
+  red: 'bg-red-01 text-white-01'
+};
+
+// 크기 스타일
+const sizeStyles = {
+  sm: 'px-[24px] py-2 max-h-[56px]',
+  md: 'max-w-[320px] max-h-[48px]',
+  lg: 'max-w-[346px] max-h-[40px]'
```

Comment:
tailwindcss에서 4px 기준입니다~
```suggestion
  sm: 'px-6 py-2 max-h-14',
  md: 'max-w-80 max-h-12',
  lg: 'max-w-[346px] max-h-10'
```